doc/api/net.markdown

```diff
@@ -723,8 +723,8 @@ Returns true if input is a version 6 IP address, otherwise returns false.
 [`pause()`]: #net_socket_pause
 [`resume()`]: #net_socket_resume
 [`server.getConnections()`]: #net_server_getconnections_callback
-[`server.listen(port, \[host\], \[backlog\], \[callback\])`]: #net_server_listen_port_hostname_backlog_callback
-[`socket.connect(options\[, connectListener\])`]: #net_socket_connect_options_connectlistener
+[`server.listen(port, host, backlog, callback)`]: #net_server_listen_port_hostname_backlog_callback
```

Comment:
The `<a id>`'s which these link to are certainly autogenerated from the doctool:

````html
<h3>
  server.listen(port[, hostname][, backlog][, callback])
  <span>
    <a class="mark" href="#net_server_listen_port_hostname_backlog_callback" id="net_server_listen_port_hostname_backlog_callback">#
    </a>
  </span>
</h3>
````
They long ids are necessary to link to different method signatures, and there can only be one id per element in HTML, so I think your suggestion is not possible.